Added documentation on how to provide access token (#8)
digital-asset/custom-views: REFERENCE.md

}
```

You can override this dispatcher through the application.conf in your application.

## Ledger API Authorization

> Client of ledger provides an access token when authorization is required by the Ledger.

Reviewer: ```suggestion
The client must provide an access token when authorization is required by the Ledger.
```

> Detail of Ledger Authorization please refer to [Ledger Authorization documentation](https://docs.daml.com/app-dev/authorization.html)

Reviewer: ```suggestion
For details of ledger authorization, please refer to [Ledger Authorization documentation](https://docs.daml.com/app-dev/authorization.html).
```

### Provide access token to custom-view library

> Application can provide an access token when setting up the clientSetting. 

Reviewer: ```suggestion
Applications can provide an access token when setting up the client. 
```

```scala
val clientSettings = GrpcClientSettings
  .connectToServiceAt(host, port.value)
  .withCallCredentials(new LedgerCallCredentials(accessToken))
val source = BatchSource.events(clientSettings)
val control = Projection.project(source, exercisedEvents)(f)
```

### Provide a newly retrieved access token when the existing one expired

> When the access token is expired, application can retrieve a new access token with the stored refresh token.

Reviewer: ```suggestion
When an access token is expired, an application can retrieve a new access token with the stored refresh token.
```

> Detail of refresh access token please refer to [Ledger auth-middleware documentation](https://docs.daml.com/tools/auth-middleware/index.html#refresh-access-token)

Reviewer: ```suggestion
For details on the refresh token, please refer to [Ledger auth-middleware documentation](https://docs.daml.com/tools/auth-middleware/index.html#refresh-access-token).
```

> Application can cancel the running projection and re-create a new one with the new access token.

Reviewer: ```suggestion
With the new access token, an application can cancel the running projection and re-create a new one using the new token.
```
